### PR TITLE
skip load/release segment when more than one delegator exist

### PR DESCRIPTION
**Pull request by weiliu1031**
Signed-off-by: Wei Liu <wei.liu@zilliz.com>
issue: #25518 #25679

### CODE REVIEW

**Comment by weiliu1031** [comment]
rerun ut

File: internal/querycoordv2/checkers/segment_checker.go [hunk 1]
```diff
@@ -112,6 +124,17 @@ func (c *SegmentChecker) checkReplica(ctx context.Context, replica *meta.Replica
 	return ret
 }
 
+func (c *SegmentChecker) checkReplicaHasRepeatedChannel(replica *meta.Replica) bool {
```
Comment (jiaoew1991) [inline internal/querycoordv2/checkers/segment_checker.go:127]: Where is this method used?

File: internal/querycoordv2/checkers/segment_checker.go [hunk 2]
```diff
@@ -83,8 +83,20 @@ func (c *SegmentChecker) Check(ctx context.Context) []task.Task {
 }
 
 func (c *SegmentChecker) checkReplica(ctx context.Context, replica *meta.Replica) []task.Task {
+	log := log.Ctx(ctx).WithRateGroup("qcv2.SegmentChecker", 1, 60).With(
+		zap.Int64("collectionID", replica.CollectionID),
+		zap.Int64("replicaID", replica.ID))
 	ret := make([]task.Task, 0)
 
+	dist := c.dist.ChannelDistManager.GetChannelDistByReplica(replica)
```
Comment (jiaoew1991) [inline internal/querycoordv2/checkers/segment_checker.go:91]: we need more comment here

**Pull request merged**